### ISSUE TITLE
A Metric Already Exists - IllegalArgumentException

### ISSUE THREAD

**Issue by psmod2** [jcustenborder/kafka-connect-rabbitmq#37]
HI,

I'm seeing the following error message after a few minutes of my RabbitMQ connector running - looks like a potential bug with the connector / I don't see any such metric options in the configuration.

Any ideas?

```
java.lang.IllegalArgumentException: A metric named 'MetricName [name=offset-commit-max-time-ms, group=connector-task-metrics, description=The maximum time in milliseconds taken by this task to commit offsets., tags={connector=RabbitMQSourceConnector, task=0}]' already exists, can't register another one.
	at org.apache.kafka.common.metrics.Metrics.registerMetric(Metrics.java:532)
	at org.apache.kafka.common.metrics.Sensor.add(Sensor.java:256)
	at org.apache.kafka.common.metrics.Sensor.add(Sensor.java:241)
	at org.apache.kafka.connect.runtime.WorkerTask$TaskMetricsGroup.<init>(WorkerTask.java:328)
	at org.apache.kafka.connect.runtime.WorkerTask.<init>(WorkerTask.java:69)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.<init>(WorkerSourceTask.java:100)
	at org.apache.kafka.connect.runtime.Worker.buildWorkerTask(Worker.java:445)
	at org.apache.kafka.connect.runtime.Worker.startTask(Worker.java:404)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.startTask(DistributedHerder.java:852)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.access$1600(DistributedHerder.java:108)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$13.call(DistributedHerder.java:866)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$13.call(DistributedHerder.java:862)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

Thanks

**Comment by jcustenborder** [comment]
This is a bug in Apache Kafka that was fixed in 1.0.0 Check out [KAFKA-6252](https://issues.apache.org/jira/browse/KAFKA-6252)

**Comment by zhipcui** [comment]
@jcustenborder 

Fix Version/s: 1.1.0, 1.0.1
